web-js/measure-last-segment/README.md

```diff
@@ -0,0 +1,10 @@
+README FOR MEASURE LAST SEGMENT
+
+Added functionality to current measurement sample where it gives a measure of the last segment.
+The sample pushes the measured values to an array and subtract the second last number from the last measurement number.
+
+What it does not do is:
+If you have completed your measurement and if you switch the unit this sample will not convert the last measured segment's unit and show the updated value.
+
+Thanks,
+Akshay Harshe
```

Comment:
would be nice if you included a [link](https://developers.arcgis.com/javascript/jssamples/widget_measurement.html) to the published measurement sample you're piggybacking off of.

also, no harm i guess, but there's probably no need to sign the README like a letter :smile: